lib/env.ts

```diff
@@ -1,10 +1,19 @@
 import Bun from "bun";
-import { object, parse, picklist } from "valibot";
+import {
+	url,
+	minLength,
+	object,
+	parse,
+	picklist,
+	string,
+	toTrimmed,
+} from "valibot";
 
 const ENV_LIST = ["development", "production"] as const;
 
 const EnvironmentSchema = object({
 	NODE_ENV: picklist(ENV_LIST, `ENV isn't in list ${ENV_LIST.join(" | ")}`),
+	DATABASE_URL: string([toTrimmed(), minLength(1), url()]),
```

Comment:
We should probably write a test to make sure this actually prevents what we want it to prevent.

Reply:
Moved to #7